week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,16 +1,23 @@
-// Add comments to explain what this function does. You're meant to use Google!
+/* Add comments to explain what this function does. You're meant to use Google!*/
 function getNumber() {
     return Math.random() * 10;
 }
 
-// Add comments to explain what this function does. You're meant to use Google!
+/*expected return: The Math.random() function returns a floating-point, pseudo-random number in the range 0 to less than 1 (inclusive of 0, but not 1) with approximately uniform distribution over that range — which you can then scale to your desired range. The implementation selects the initial seed to the random number generation algorithm; it cannot be chosen or reset by the user.
+-source - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random */
+
+/* Add comments to explain what this function does. You're meant to use Google!*/
```

Comment:
Well done!

As a note, it is common to place any comments documenting your functions, just above each function, e.g.:

```javascript

/**
 * This method does something very cool!
 * A brief descriptions of the parameters: firstWord, secondWord
 * And here, a brief description on what this method returns.
*/
function doSomethingVeryCool(firstWord, secondWord) {
  someMagic();
  return 1 + 2 + 3;
}
```

This way you'll always know which comment refers to which function :)